skyfield/data/mpc.py

```diff
@@ -226,6 +227,31 @@ def comet_orbit(row, ts, gm_km3_s2):
     comet._rotation = inertial_frames['ECLIPJ2000'].T
     return comet
 
+def comet_orbits(rows, ts, gm_km3_s2):
```

Comment:
This reads so cleanly for a vectorized routine! Could you add a `_` in front of the name for now, until we can further test it?